modules/util.lua

```diff
@@ -6,20 +6,30 @@
 local pairs = pairs
 local type = type
 local string = require "string"
+local cjson = require "cjson"
 
 local M = {}
 setfenv(1, M) -- Remove external access to contain everything in the module
 
 -- Flattens a Lua table so it can be encoded as a protobuf fields object.
-function table_to_fields(t, fields, parent)
+function table_to_fields(t, fields, parent, char, max_depth)
```

Comment:
ah, ok, that makes sense re: msg field schema, thanks. i'll go get the tests added. :)